example-client/example-client.cabal

```diff
@@ -12,7 +12,7 @@ build-type:          Simple
 cabal-version:       >=1.10
 
 tested-with:
-  GHC==9.2.1, GHC==9.0.2,
+  GHC==9.4.1, GHC==9.2.1, GHC==9.0.2,
```

Comment:
Can you bump 9.2.1 to 9.2.4 as well?

Reply:
With pleasure! (Just yesterday learned how to do group editing in one Emacs buffer 🤩) Done.